imports/plugins/custom/notifications/server/methods/notifications.js

```diff
@@ -0,0 +1,92 @@
+import { Meteor } from "meteor/meteor";
+import { check, Match } from "meteor/check";
+import { Reaction, Logger } from "/server/api";
+import { Notifications, Packages } from "/lib/collections";
+
+/**
+ * Reaction Notification methods
+ */
+Meteor.methods({
+  /**
+  * notification/send
+  * @summary This send a notification to a user
+  * @param {String} userId - The user
+  * @param {String} type - The type of Notification
+  * @param {String} url - url link
+  * @param {Boolean} sms - sms enabled check.
+  * @param {String} details - details of the Notification
+  * @return {Object} returns result
+  */
+  "notification/send": function (userId, type, url, sms, details) {
+    check(userId, String);
+    check(type, String);
+    check(sms, Boolean);
+    check(details, Match.OptionalOrNull(String));
+    check(url, String);
+
+    const values = {};
+    const types = {
+      orderCancelled: "Your order was cancelled.",
+      newOrder: "Your order is being processed.",
+      topWallet: "Your wallet has been credited.",
```

Comment:
The messages are predefined for sending the messages a user should expect. In this case, the message comes in handy so as to prevent lengthy messages.